### DISCUSSION TITLE
How to display text as a smooth slide?

### DISCUSSION THREAD

**Discussion by marc-hg** [pybricks/support#1252] [Q&A]
In the official lego app, the block to display text shows each letter smoothly going from right to left

Is there any way I could do this with pybricks? display.text just shows the characters blinking

**Comment by laurensvalk** [comment]
There's no builtin function for this, but you can get creative with the animation or multitasking if you like :smile:

**Reply by EclecticInventor** [reply]
I was also looking to see if there was an easy way to do this, without creating a billion Matrices.

**Comment by laurensvalk** [comment]
You can find two examples in this discussion: https://github.com/orgs/pybricks/discussions/1314#discussioncomment-7763953

**Reply by NiHoffmann** [reply]
Is there a way to access the Display Character Set for this ? 
So i don't have to have define all letters/numbers again if i want to display strings that aren't pre-definded ?

**Reply by laurensvalk** [reply]
Not at the moment, but I suppose we could expose that via the `Icon` parameter class. I'll open a feature request for that.

**Reply by NiHoffmann** [reply]
thanks a lot that would be a neat addition
